new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -126,60 +178,59 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
     logger.info("NGINX http node port: {0}", nodeportshttp);
   }
 
+  @AfterAll
+  public void tearDownAll() {
+    // uninstall NGINX release
+    if (nginxHelmParams != null) {
+      assertThat(uninstallNginx(nginxHelmParams))
+          .as("Test uninstallNginx returns true")
+          .withFailMessage("uninstallNginx() did not return true")
+          .isTrue();
+    }
+  }
+
   /**
    * Create operator and verify it is deployed successfully.
    * Create a custom domain resource and verify all server pods in the domain were created and ready.
    * Verify NGINX can access the sample app from all managed servers in the domain.
-   * Delete operator.
+   * Uninstall operator.
```

Comment:
Create Operator -->  Install the Operator successfully. 

Create a custom domain resource and verify all server pods in the domain were created and ready ---> Deploy a custom domain resource and verify all the servers pod are ready.  Deploy a sample web application to WebLogic Cluster.
sample app --> sample application
Uninstall -->  Uninstall the Operator Helm Chart.

Reply:
done